bio_hansel/scripts/get_sequence.py

```diff
@@ -0,0 +1,69 @@
+from typing import Dict
+
+import pandas as pd
+
+from Bio import SeqIO
+
+
+def get_sequences(output_directory: str, reference_genome_path: str,
+                  results_dict: Dict[str, pd.DataFrame], schema_name: str) -> None:
+    """Collects the sequences from the reference genome by going through the list of dataframes and outputting the
+    associated sequences at that SNV location
+
+    Args:
+        output_directory:directory where the schema would be located as indicated by the user
+        results_dict: specifies the list of genomes and their associated group
+        reference_genome_path: file path to where the reference genome is located
+        schema_name: the name of the output schema file
+
+    Returns:
+        Creates schema file in the output directory
+
+
+    """
+    for key, value in results_dict.items():
+        group = key
+        max_sequence = value.loc[value['POS'].idxmax()]
+        gb_file = reference_genome_path
```

Comment:
Does `reference_genome_path` need to be assigned to `gb_file`?

Reply:
this has been fixed 4277dffbf5dc652244a1425057a3cee485b4b4e1